### DISCUSSION TITLE
How to put markdown and relative jpg in one directory when use docs

### DISCUSSION THREAD

**Discussion by cocobase** [facebook/docusaurus#7442] [General]
1. I setup a new docs directory called working
2. when I put markdown in working directory, it works well
3. I have a markdown file with serveral relative image file
4. I create a directory called [abc] and put all these files into [abc]
5. I can't see my file in docusaurus website 

what I should do to fix this ? 
It seemed when I use blog , I can use a directory to put all relative files together.

**Comment by Josh-Cena** [comment]
Please provide your directory structure and where you expect your page to be found in your site.

**Reply by cocobase** [reply]
## Web page 
![截屏2022-05-19 上午10 45 07](https://user-images.githubusercontent.com/60334709/169193549-f7c226f8-34ae-49a5-b25e-442221d67e51.png)

## My directory structure is here
<img width="354" alt="截屏2022-05-19 上午10 42 28" src="https://user-images.githubusercontent.com/60334709/169193536-1d9481b5-2d17-4e29-87ec-d5c36eded756.png">

**Reply by Josh-Cena** [reply]
It's a bug that has been fixed in canary. See https://github.com/facebook/docusaurus/issues/7348

**Reply by cocobase** [reply]
According #7348 issue's suggestion. I changed to Canary release

It works very well now.

Thank you for your advision.
